addons/mail/static/src/xml/thread.xml

```diff
@@ -88,7 +88,7 @@
                 </script>
             </head>
             <body onload='onload_img()'>
-                <img t-att-src='src' />
+                <img t-att-src='src' alt=""/>
```

Comment:
What's the difference between no alt and empty alt, from a screenreader POV?

Reply:
With no alt, the screen reader says there is an image with no description. With empty alt, it ignores the image because there is no need of description.